src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -1020,6 +1020,10 @@
       <field ref="remarks" in-xml="WITH_WRAPPER"/>
     </model>
     <constraint>
+      <allowed-values target="link/@rel" allow-other="yes">
+        <!-- TODO: More work needs to be done around link enforcement constraints and requires Metaschema work on recurse-depth-first() recursive index searching. -->
+        <enum value="imported-from">The hyperlink identifies a URI pointing to the <code>component</code> in a <code>component-definition</code> that originally defined the <code>component</code>.</enum>
```

Comment:
@david-waltermire-nist -- I am thinking that using "that originally **defined** the <code>component</code>" might be misleading since the CDef suggests control implementations, does not **define** them as in **deciding** them. You made recently an important correction/clarification for the CDef.

Reply:
Please propose a wording change.

Reply:
I'd like to merge this on 8/30. If @iMichaela doesn't provide a wording change suggestion, I am good with keeping this with a minor tweek.

```suggestion
        <enum value="imported-from">The hyperlink identifies a URI pointing to the <code>component</code> in a <code>component-definition</code> that originally described the <code>component</code> this component was based on.</enum>
```

Reply:
Suggestion merged, will rebase and squash in a little bit.